.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -262,7 +262,8 @@ jobs:
               -DKokkos_ENABLE_SERIAL=ON \
               -DKokkos_ENABLE_OPENMP=ON \
               -DENABLE_WARNINGS=OFF \
-              -DENABLE_OPENQASM=ON
```

Comment:
You can remove line 261 and update line 257 as follows:
```
              -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=$GITHUB_WORKSPACE/runtime-build/lib \
```

Reply:
Why are we removing the git tag specification?

Reply:
yeah, I've added it and realized this's not make commands :smile: I thought that I updated my code comment before posting. Anyway, I've edited the suggestion